src/Resources/translations/VerifyEmailBundle+intl-icu.en.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/*
+ * This file is part of the SymfonyCasts VerifyEmailBundle package.
+ * Copyright (c) SymfonyCasts <https://symfonycasts.com/>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return [
+    '%count% month|%count% months' => '{count, plural, one {# month } other {# months }}',
+    '%count% day|%count% days' => '{count, plural, one {# day } other {# days }}',
+    '%count% hour|%count% hours' => '{count, plural, one {# hour } other {# hours }}',
+    '%count% minute|%count% minutes' => '{count, plural, one {# minute } other {# minutes }}',
+];
```

Comment:
I think `.xlf` files are the "standard" format for translations in a bundle

Reply:
done